### ISSUE TITLE
Using ldaps and certificates

### ISSUE THREAD

**Issue by mauritsvanrees** [bluedynamics/node.ext.ldap#28]
Has anyone used `pas.plugins.ldap` / `node.ext.ldap` successfully with the `ldaps` protocol? I have a configuration that works with `ldap://some-host:389` but fails with `ldaps://some-host:636`.  With that we get in the logs:

```
WARNING pas.plugins.ldap
LDAPError in authenticateCredentials ->
{'info': 'TLS: hostname does not match CN in peer certificate',
'desc': "Can't contact LDAP server"}
```

It could be that the certificate on the LDAP server is really wrong. But I also think it is a self-signed certificate, so `pas.plugins.ldap` may have trouble checking it.
Indeed when I add `server.ignore_cert=True` in `ldapsettings.xml` it works. I see in the changelog that this has been added especially for self-signed certificates.

Did you ever get certificate checking to work properly with a self-signed certificate? Some environment option maybe that says where to find a certificate?

Possibly some code from this [Django snippet](https://djangosnippets.org/snippets/901/) may help.

**Comment by rnixx** [comment]
Please try to set ``tls_cacertfile`` on ``LDAPServerProperties``. This property is not exposed to the properties form in ``pas.plugins.ldap`` so you need to hook in there somehow.

https://github.com/collective/pas.plugins.ldap/blob/master/src/pas/plugins/ldap/properties.py#L105

**Comment by rnixx** [comment]
@mauritsvanrees any new insight on this issue?

**Comment by rnixx** [comment]
@mauritsvanrees is this still an issue for you?

**Comment by mauritsvanrees** [comment]
In `ldapsettings.xml` I have `<element key="server.ignore_cert" type="bool">True</element>`. That does not solve the issue, but works around it.

It is for a client where I need a VPN to even access the site, and I don't think the VPN currently works. 
The LDAP server is internal too, which is probably true for at least 99% of Plone Sites that use LDAP. So fine to either use ldap, or ldaps without checking the certificate.

In short: no, it is not really an issue anymore. I will close the bug.
